ProcessMaker/processmaker
# Redesign notification bell popover

## Issue & Reproduction Steps
See
- https://processmaker.atlassian.net/browse/FOUR-9710
- https://processmaker.atlassian.net/browse/FOUR-10163
- PRD: https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3148578817/Notifications+Overhaul

Requires https://github.com/ProcessMaker/package-comments/pull/89

ci:next
ci:deploy
ci:package-comments:feature/FOUR-9709
.

## Solution
- Redesign the bell icon popover window
- Standardize notification messages
- Remove the items once viewed

## How to Test
Create a number of notifications. For example
- Comments
- Process started
- Task assigned

You can put viewed messages back in the popover by going to the `/notifications` page and clicking the mail icon to un-read

Ensure it matches the Figma in the [PRD](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3148578817/Notifications+Overhaul)

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-9710
- https://processmaker.atlassian.net/browse/FOUR-10163
- https://github.com/ProcessMaker/package-comments/pull/89

## Code Review Checklist
